sdks/python/apache_beam/transforms/userstate.py

```diff
@@ -357,6 +357,10 @@ def prefetch(self):
     # The default implementation here does nothing.
     pass
 
+  def finalize(self):
```

Comment:
@robertwb do you see any concerns with adding a top level `finalize` method here?

Reply:
This is fine .